lib/ancestry/has_ancestry.rb

```diff
@@ -31,7 +36,7 @@ def has_ancestry options = {}
     self.touch_ancestors = options[:touch] || false
 
     # Validate format of ancestry column value
-    validates_format_of ancestry_column, :with => Ancestry::ANCESTRY_PATTERN, :allow_nil => true
+    validates_format_of ancestry_column, :with => ancestry_pattern, :allow_nil => true
```

Comment:
yea, if this were `:ancestry_pattern` then it would call each time.

since this is `ancestry_pattern` pretty sure it evaluates only once before even going into `validates_format_of`